src/core/DataFile.cpp

```diff
@@ -868,10 +869,21 @@ void DataFile::upgrade_1_1_91()
 	for( int i = 0; !list.item( i ).isNull(); ++i )
 	{
 		QDomElement el = list.item( i ).toElement();
-		if ( el.attribute( "name" ) == "plugin" && el.attribute( "value" ) == "vocoder-lmms" ) {
+		if( el.attribute( "name" ) == "plugin" && el.attribute( "value" ) == "vocoder-lmms" ) {
 			el.setAttribute( "value", "vocoder" );
 		}
 	}
+
+	list = elementsByTagName( "crossoevereqcontrols" );
+	for( int i = 0; !list.item( i ).isNull(); ++i )
+	{
+		QDomElement el = list.item( i ).toElement();
+		// invert the mute LEDs
+		for( int d = 1; d <= 4; ++d ){
```

Comment:
Yeah, we use that logic all [over the place](https://github.com/lmms/lmms/blob/master/src/core/DataFile.cpp#L1017) and if it were to fail, we'd have major problems.   The general logic is `while (true) { if !bar.hasAttribute("foo" + (i++)) break; else continuishlogic; }`